### DISCUSSION TITLE
Spfx form customizer using dynamic form (react)

### DISCUSSION THREAD

**Discussion by DeathMaul23** [SharePoint/sp-dev-docs#8595] [Q&A] [answered]
Hi, I tried using pnp dynamic forms [Dynamic form documentation](https://pnp.github.io/sp-dev-fx-controls-react/controls/DynamicForm/) for saving lists which works fine, but saving doc sets I get this error 
![MicrosoftTeams-image](https://user-images.githubusercontent.com/33746224/202705494-1e4efdc6-91ee-41b7-be9a-b5191e460a21.png)

Any other methods for saving docsets than using 
`  private _onSave = (listItemData: any, listItem?: IItem): void => {
    this.formSaved();
  }`

I used this repo as sample [Conference-Sample](https://github.com/PiaSys/Conferences-Samples/tree/master/spfx-pnp-controls)

**Comment by AJIXuMuK** [comment, marked as answer]
The latest release of PnP Controls adds support for doc sets creation.
From SPFx Core perspective - a developer is responsible for the logic of creating an item.
